### PR TITLE
fix(cwts): Display CWTS on signup page for Fenix and iOS

### LINKED ISSUE #4603: CWTS is not displayed in signup page on Android and iOS
**Build:**
Stage [Train 1.163.0]

**Affected platforms:**
Android 9
iOS 13.1.2

**Steps to reproduce:**
1. Launch Firefox app.
2. Go to menu -> settings -> SIGN IN

**Expected results:**
CWTS section is successfully displayed in signup page for Android and iOS users.

**Actual Results:**
CWTS section is not displayed in registration form. 

Screenshot_10

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1393)
┆Issue Number: FXA-1393

### PR DESCRIPTION
**Pull request by vladikoff**
Also, actually run the CWTS Sign up Tests

Fixes #4603

@vbudhram r?

### CODE REVIEW

File: packages/fxa-content-server/app/tests/test_start.js [hunk 1]
```diff
@@ -168,6 +168,7 @@ require('./spec/views/mixins/account-reset-mixin');
 require('./spec/views/mixins/avatar-mixin');
 require('./spec/views/mixins/back-mixin');
 require('./spec/views/mixins/cached-credentials-mixin');
+require('./spec/views/mixins/cwts-on-signup-password');
```
Comment (vladikoff) [inline packages/fxa-content-server/app/tests/test_start.js:171]: 👀

**Comment by vladikoff** [comment]
image

File: packages/fxa-content-server/app/tests/test_start.js (hunk 1 above)
Comment (vbudhram) [reply packages/fxa-content-server/app/tests/test_start.js:171]: 🙇🏽‍♂️

**Pull request merged**